sys/include/ztimer.h

```diff
@@ -301,12 +301,26 @@ struct ztimer_base {
     uint32_t offset;            /**< offset from last timer in list */
 };
 
+/**
+ * @defgroup   sys_ztimer_now64 ztimer_now64
+ * @brief 64-bit timestamp support
+ *
+ * @deprecated use @ref ztimer_now() returning uint32_t or alternatively use
+ *             module `ztimer64` with @ref ztimer64_now() returning uint64_t.
+ *             Will be removed after 2022.10 release.
+ */
+/**
+ * @typedef ztimer_now_t
```

Comment:
Might not be needed, but not blocking for this.

Reply:
I think it is easier to read this way (i think you suggest the `@typedef ...` might not be needed)